dockerfiles/jupyter/README.ipynb

```diff
@@ -48,22 +90,30 @@
     "- [**Train, Compare, and Register Models**](./tutorial/02-model-training.ipynb)\n",
     "- [**Serving ML/DL models**](./tutorial/03-model-serving.ipynb)\n",
     "- [**Projects and Automated ML Pipeline**](./tutorial/04-pipeline.ipynb)\n",
-    "- [**Real-time Monitoring & Drift Detection (currently require Iguazio cluster)**](./tutorial/05-model-monitoring.ipynb)\n",
-    "- [**Add MLOps to existing code**](./tutorial/06-add-mlops-to-code.ipynb)\n",
-    "- [**Batch Inference and Drift Detection**](./tutorial/07-batch-infer.ipynb)\n",
-    "\n",
     "\n",
     "You can find different end to end demos in under the [**demos folder**](./demos/README.md).\n",
+    "\n",
+    "> **Please note** - All the demos that requires the addional components will not work with the default installation and you should enable those component before running the examples.\n",
     "\n"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "9ce594a4-2515-468e-9283-b39530d1ee32",
+   "cell_type": "markdown",
+   "id": "37671239",
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "source": [
+    "## Configure Your Environment to your IDE\n",
+    "\n",
+    "This Jupyter Notebook server is designed to work with the different options, it works out of the box with the **local** deployment mode, and you can connect your MLRun CE with your IDE (VSCode or PyCharm).<br> \n",
```

Comment:
"This Jupyter Notebook server is designed to work with the different options. It works out of the box with the **local** deployment mode, and you can connect your MLRun CE with your IDE (VSCode or PyCharm).<br> \n",